### ISSUE TITLE
Keyboard Shortcuts don't work

### ISSUE THREAD

**Issue by rickhumphries** [elmsln/elmsln#1498]
Bug:
- v0.6.0

I can 'alt+shift' to see the keyboard shortcuts:
![image](https://cloud.githubusercontent.com/assets/4973810/20504169/0d4ccbea-b03e-11e6-9719-3d761fbe3459.png)

 But when I type any of the letters nothing happens

**Comment by rickhumphries** [comment]
I should mention though that the 'scroll up' 'scroll down' 'next page' and 'previous page' keyboard shortcuts work fine though

**Comment by btopro** [comment]
do you mean just the `o` or all of them? you have to hold the shortcut and then press the other key. so like `alt + shift + h` will trigger the help. `o` does appear to have an issue in my testing

**Comment by rickhumphries** [comment]
Ahh, OK - I didn't know it was `alt + shift + whatever`. That works for me now. 

But you're right. `o` doesn't work for me either

**Comment by btopro** [comment]
yeah weird things broken in keyboard and voice commands when moving to materialize that have still elluded me. closing menus w/ your voice and keyboard nav w/ random keys; I just have to make their trigger operate differently
